application/src/utils/i18next.ts

```diff
@@ -23,7 +23,8 @@ i18n
       en: { translation: translationENG },
       ru: { translation: translationRU }
     },
-    react: { wait: false }
+    // react: { wait: false }
+    react: { useSuspense: false }
```

Comment:
Just be aware that choosing not to use Suspense has its implications. You have to write checks to handle the 'not ready' state ie: have a loading component render when state is not ready and your component render when the state is ready. Not doing so will result in rendering your translations before they loaded.
![image](https://user-images.githubusercontent.com/57001255/146517226-8db35253-dfc8-4ef6-8d53-e39a12595710.png)

Reply:
i fixed, but i did this because i had special warning from tests. console.warn
    react-i18next:: It seems you are still using the old wait option, you may migrate to the new useSuspense behaviour. And i find only 1 working decision:react: { useSuspense: false }